src/middleware/errorHandler.js

```diff
@@ -133,9 +133,7 @@ const filterSecrets = (error, req, res, next) => {
 		req.originalUrl = filterQuery(req.originalUrl);
 		req.body = filter(req.body);
 		error.data = filter(error.data);
-		if (error.catchedError && error.catchedError.options && error.catchedError.options.form) {
-			error.catchedError.options.form = filter(error.catchedError.options.form);
-		}
+		error = filter(error);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to function parameter 'error'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5696184)